db/__init__.py

```diff
@@ -1,5 +1,17 @@
 from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import MetaData
+
+convention = {
+    "ix": "ix_%(column_0_label)s",
```

Comment:
dict keys should be single quote: https://www.askpython.com/python/string/difference-between-single-and-double-quotes-in-python#:~:text=Single%20quotes%20are%20used%20for%20regular%20expressions%2C%20dict%20keys%20or%20SQL.